### ISSUE TITLE
Support for Mac OS aliases?

### ISSUE THREAD

**Issue by ryanpcmcquen** [deathau/markdownload#62]
I am attempting to use a Mac OS alias for my CLIPS folder so I can integrate the extension with Obsidian. But the downloads fail when I do this.

<img width="548" alt="Screen Shot 2021-02-25 at 8 55 20 AM" src="https://user-images.githubusercontent.com/772937/109187918-3590d680-7747-11eb-8ca6-b7c535c95ae7.png">

<img width="552" alt="Screen Shot 2021-02-25 at 8 56 21 AM" src="https://user-images.githubusercontent.com/772937/109188051-59ecb300-7747-11eb-82c0-30fc030ba962.png">

Is there some kind of trick or workaround?

**Comment by deathau** [comment]
I don't think it will work with an alias like that, but you might be able to get it to work with a [Symbolic Link](https://www.howtogeek.com/297721/how-to-create-and-use-symbolic-links-aka-symlinks-on-a-mac/) which works a bit differently (Disclaimer: I have not attempted this myself)

**Comment by deathau** [comment]
I'll keep this issue open though, so I can remind myself to look into aliases and see if there's anything I can do to get them to work. 

I doubt it, but it can't hurt to do some research

**Comment by ryanpcmcquen** [comment]
The symlink worked!
